htdocs/langs/en_US/agenda.lang

```diff
@@ -104,6 +104,14 @@ TICKET_MODIFYInDolibarr=Ticket %s modified
 TICKET_ASSIGNEDInDolibarr=Ticket %s assigned
 TICKET_CLOSEInDolibarr=Ticket %s closed
 TICKET_DELETEInDolibarr=Ticket %s deleted
+BOM_VALIDATEInDolibarr=BOM validated	
+BOM_UNVALIDATEInDolibarr=BOM unvalidated	
+BOM_CLOSEInDolibarr=BOM disabled	
+BOM_REOPENInDolibarr=BOM reopen	
+BOM_DELETEInDolibarr=BOM deleted	
+MO_VALIDATEInDolibarr=MO validated	
+MO_PRODUCEDInDolibarr=MO disabled	
```

Comment:
@eldy I took definition from agenda config
MO_PRODUCEDInDolibarr=MO disabled is right?